### PR TITLE
Sqlite handler

### PR DESCRIPTION
**Pull request by Chitti-Ankith**
This diff adds support for SQLite database handler. Closed the previous commit since it was pushed onto the staging branch instead of master.

### CODE REVIEW

**Comment by Chitti-Ankith** [comment]
@xzdandy can you please take a look at the diff now

**Comment by xzdandy** [comment]
> @xzdandy can you please take a look at the diff now

Hi Chitti-Ankith, Thanks for adding the sqlite3 integration! Please check https://evadb.readthedocs.io/en/stable/source/dev-guide/contribute/submit-pr.html, PRs should be syncing and merging with staging branch.

File: evadb/third_party/databases/sqlite/sqlite_handler.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    def __init__(self, name: str, **kwargs):
+        """
+        Initialize the handler.
+        Args:
+            name (str): name of the DB handler instance
+            **kwargs: arbitrary keyword arguments for establishing the connection.
+        """
+        super().__init__(name)
+        self.database = kwargs.get("database")
+
```
Comment (gaurav274) [inline evadb/third_party/databases/sqlite/sqlite_handler.py:36]: Add `self.connection = None`

File: evadb/third_party/databases/sqlite/sqlite_handler.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+            DBHandlerResponse
+        """
+        if not self.connection:
+            return DBHandlerResponse(data=None, error="Not connected to the database.")
+        """
+        SQLite does not provide an in-built way to get the column names using a SELECT statement.
+        Hence we have to use the PRAGMA command and filter the required columns.
+        """
+        try:
+            query = f"PRAGMA table_info('{table_name}')"
```
Comment (gaurav274) [inline evadb/third_party/databases/sqlite/sqlite_handler.py:100]: It should also return the column_types. See the abstract function for reference.

File: evadb/third_party/databases/sqlite/sqlite_handler.py (hunk 1 above)
Comment (Chitti-Ankith) [reply evadb/third_party/databases/sqlite/sqlite_handler.py:36]: Done

File: evadb/third_party/databases/sqlite/sqlite_handler.py (hunk 2 above)
Comment (Chitti-Ankith) [reply evadb/third_party/databases/sqlite/sqlite_handler.py:100]: Done

**Pull request closed without merging**